### ISSUE TITLE
Too many IllegalStateExceptions

### ISSUE THREAD

**Issue by paour** [ACRA/acra#3]
It's laudable that ACRA would report incorrect configuration -- during debug. At runtime, ACRA should fail as silently and as gracefully as possible, not crash the application.

Case in point: I'm calling ACRA.init(this) only once in my application object's onCreate() method, and I'm getting this error:

`java.lang.RuntimeException: Unable to start receiver com.frogsparks.mytrails.LicenseInstalled: java.lang.RuntimeException: Unable to create application com.frogsparks.mytrails.MyTrailsApp: java.lang.IllegalStateException: ACRA#init called more than once
        at android.app.ActivityThread.handleReceiver(ActivityThread.java:1809)
        at android.app.ActivityThread.access$2400(ActivityThread.java:117)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:985)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:130)
        at android.app.ActivityThread.main(ActivityThread.java:3687)
        at java.lang.reflect.Method.invokeNative(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:507)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:842)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
        at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.RuntimeException: Unable to create application com.frogsparks.mytrails.MyTrailsApp: java.lang.IllegalStateException: ACRA#init called more than once
        at android.app.LoadedApk.makeApplication(LoadedApk.java:479)
        at android.app.ActivityThread.handleReceiver(ActivityThread.java:1783)
        ... 10 more
Caused by: java.lang.IllegalStateException: ACRA#init called more than once
        at org.acra.ACRA.a(SourceFile:118)
        at com.frogsparks.mytrails.MyTrailsApp.onCreate(SourceFile:135)
        at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:969)
        at android.app.LoadedApk.makeApplication(LoadedApk.java:476)
        ... 11 more`

It would be nice to have a switch for strict mode (throws IllegalStateException) and shipping/non-strict (does not throw exceptions unless absolutely necessary).
